src/main/java/mate/academy/ConnectionUtil.java

```diff
@@ -0,0 +1,54 @@
+package mate.academy;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/learning";
+    private static final Properties DB_PROPERTIES;
+    private static final String INIT_FILE_PATH = "src/main/resourses/init_db.sql";
+    private static String init_script;
+
+    private ConnectionUtil() {
+
```

Comment:
to make it 100% uncreatable
```suggestion
        throw new UnsupportedOperationException("Util class");
```